packages/android_intent/test/android_intent_test.dart

```diff
@@ -140,6 +140,52 @@ void main() {
         verifyZeroInteractions(mockChannel);
       });
     });
+
+    group('showChooser', () {
```

Comment:
Related to a couple of my earlier questions, I think that tests should probably exist in this group based on a situation where no target activity is found, and any other situation that would result in a chooser not being shown.